### ISSUE TITLE
Blokada turns itself off when Wi-fi hotspot is on

### ISSUE THREAD

**Issue by fxsmthn** [blokadaorg/blokada#86]
This functionality is bit annoying because I'm often sharing Wi-fi hotspot to a friend or to my PC, so ads are then shown on my phone. I'm curious why Blokada works this way  (turning itself off if Wi-fi tethering is on), when DNS66 continues working even with hotspot on. Besides this problem, Blokada seems to work very well on Oreo, thanks a lot for this great application Blokada team!

Device: Nokia 8
Android: 8.0.0
Blokada: 2.2.3 F-Droid

**Comment by kar** [comment]
Try turning off "online only" mode

**Comment by fxsmthn** [comment]
It doesn't seem to make a difference to me, Blokada disables itself when it detects Hotspot being on regardless of the state of the Online only toggle.

I'm just really curious if there was some issue caused with Hotspot and Blokada running simultaneously. My thought goes to American carriers, that are stingy regarding Hotspot and disable it in carrier branded phones...

This comment I found from code: "// Auto off in case of no connectivity or tethering enabled, and auto on once connected" and then the code.

I found a "fix" for myself too: I used certain "unsafe application" to deprive Blokada the ACCESS_WIFI_STATE permission and now it works while other devices are connected because Blokada doesn't know Wi-fi hotspot is on! Not very elegant solution, but so far, no issues for me.
